.github/workflows/grid_client_tests.yml

```diff
@@ -23,6 +23,7 @@ jobs:
       RMB_PROXY: true
       STORE_SECRET: secret
       MNEMONIC: ${{ secrets.MNEMONIC }}
+      SKIP_PROPS_VALIDATION: true
```

Comment:
please remove this and update the test expectation to throw when setting the value on the object

Reply:
Fixed
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/31689104/21941a37-1dc1-41ee-a2eb-495b84831733)